### ISSUE TITLE
Having trouble with nuxtjs

### ISSUE THREAD

**Issue by rolandszpond** [krystalcampioni/vue-hotel-datepicker#51]
I'm having trouble getting this to work with Nuxtjs generated sites... 
the error that I'm getting is "document is not defined" due to server side rendering. I've tried everything that nuxt community suggests to get the code to pull only on the client side, but I have no luck...

i.e.
nuxt.config.js
```
modules.exports = {
	plugins: [
		{
			src: 'vue-hotel-datepicker',
			ssr: false
		}
	]
}
```

or in a vue component:
```
if (process.browser) {
	require('vue-hotel-datepicker')
}

export default {
	
}
```

**Comment by rolandszpond** [comment]
I just discovered the <no-ssr> tag buried deep into one of their examples on their github: https://github.com/nuxt/nuxt.js/blob/dev/examples/no-ssr/pages/index.vue, which seems to let it work after generating and publishing pages to hosting. But, I don't think this is a "real" solution, I think making this plugin available for srr for using with nuxt should be implemented. I will create a PR once i'm about to get around to it, but for now, wrapping it with the <no-ssr> tags works

**Comment by antony** [comment]
Yep, just ran into the same issue. The plugin is awesome, but it really needs SSR support. Did you ever get around to a PR?

**Comment by rolandszpond** [comment]
@antony try wrapping the component with <no-ssr> tags in your html

**Comment by antony** [comment]
@rolandszpond that's not a solution to SSR, that's just circumventing it.

**Comment by rolandszpond** [comment]
@antony I have not, unfortunately. The project that I was gonna apply it to switched directions in how we were gonna use a datepicker and we are using a different component for that.

**Comment by Codefa** [comment]
@rolandszpond It works fine for me its kinda a late anwer anyway this answer will help others might looking for the same issue with nuxt

Just import the component like in vue on your page and add the code within` <no-ssr></no-ssr>`  
like this
```
<no-ssr>
    <HotelDatePicker format="DD/MM/YYYY">
    </HotelDatePicker>
</no-ssr>

**Comment by rolandszpond** [comment]
@Codefa Yes, <no-ssr> works great. As far as the component having to be server side rendered, after some thought with it, I don't think there's a need for a datepicker being server rendered. The benefit for SSR is for search clients being able to crawl your site data and you wouldn't be gaining anything for them to crawl just a datepicker.

**Comment by Codefa** [comment]
@rolandszpond absolutely, after fetching the date from the picker we can make it as a Server Side Rendered.., my thoughts..
